lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -309,7 +309,7 @@ bool UsdAttributes::hasAttribute(const std::string& name) const
 
 #ifdef UFE_V4_FEATURES_AVAILABLE
 #if (UFE_PREVIEW_VERSION_NUM >= 4024)
-Ufe::AddAttributeCommand::Ptr
+Ufe::AddAttributeUndoableCommand::Ptr
```

Comment:
I think it was mentioned in other comments. You'll need an extra ifdef here for >= 4033 check to use "AddAttributeUndoableCommand" (keeping AddAttibuteCommand for >= 4024).

Reply:
yes, I should have added the necessary `ifdef` for the new typedef

Reply:
Its okay, but personally I would have just wrapped the return value in the if/else